pkgs/development/libraries/vulkan-loader/default.nix

```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
     sha256 = "0zhrwj1gi90x2w8gaaaw5h4b969a8gfy244kn0drrplhhb1nqz3b";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ pkgconfig patchelf ];
```

Comment:
`patchelf` is included in `stdenv` for Linux by default so no need to add it here.